glib/src/subclass/object.rs

```diff
@@ -362,7 +362,7 @@ mod test {
                             .get()
                             .expect("type conformity checked by 'Object::set_property'");
                         self.name.replace(name);
-                        obj.emit_by_name("name-changed", &[&*self.name.borrow()]);
+                        obj.emit_by_name::<()>("name-changed", &[&*self.name.borrow()]);
```

Comment:
This is a bit unfortunate: if you just don't use the return value the compiler doesn't know what type to expect.